README.md

```diff
@@ -97,6 +97,7 @@ The example above gives you a bare-bones pipeline, and there may be Jenkinsfile
 * [ConfirmApplyPlugin](./docs/ConfirmApplyPlugin.md): pause and review the plan, before applying any changes.
 * [ConditionalApplyPlugin](./docs/ConditionalApplyPlugin.md): only allow apply on master branch.
 * [DefaultEnvironmentPlugin](./docs/DefaultEnvironmentPlugin.md): automatically set `TF_VAR_environment` variable.
+* [TargetPlugin](./docs/TargetPlugin.md): set `-target` parameter for terraform plan and apply.
```

Comment:
For the "Default Plugins" category - I had intended it to mean, "Plugins that are enabled with you doing anything."   That's not the case for TargetPlugin is it?  I hadn't planned on having it enabled by default.

Reply:
Oh Sorry, I didn't know exactly what you mean by "Default Plugins". I would say this is NOT a default plugin.

Reply:
Brainstorming, and asking your opinion here. 

I think this could go under "Other" OR it seems like we're building out a set of plugins that are kind of terraform-centric.  The TargetPlugin, the DirectoryPlugin, these are augmenting/configuring the behavior of terraform commands themselves. Maybe this warrants it's own category?

Either way, this is no big deal, but it's worth continuing to shape the organization of the project/plugins as a whole.

Reply:
I moved it under "Other" for now